gradle-plugins/gradle/libs.versions.toml

```diff
@@ -0,0 +1,17 @@
+[versions]
+kotlin = "1.9.0"
+gradle-download-plugin = "5.5.0"
+kotlinx-serialization = "1.2.1"
```

Comment:
latest kotlinx-serialization version is 1.6.0
Can we consider upgrade? Or better to keep as it was been before?

Reply:
Oh, I saw another PR. (https://github.com/JetBrains/compose-multiplatform/pull/3607)
That's fine to keep 1.2.1 for now